go/vt/wrangler/reparent.go

```diff
@@ -616,7 +647,10 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 			if err1 := wr.tmc.UndoDemoteMaster(undoCtx, oldMasterTabletInfo.Tablet); err1 != nil {
 				log.Warningf("Encountered error %v while trying to undo DemoteMaster", err1)
 			}
-			return fmt.Errorf("master-elect tablet %v failed to catch up with replication or be upgraded to master: %v", masterElectTabletAliasStr, err)
+			return fmt.Errorf("master-elect tablet %v failed to be upgraded to master: %v", masterElectTabletAliasStr, err)
+		} else if ctx.Err() != nil && ctx.Err() == context.DeadlineExceeded {
```

Comment:
This doesn't need to be an else, does it?

Reply:
Fixed.